.github/workflows/ci.yml

```diff
@@ -63,3 +63,29 @@ jobs:
             ci build verify-build
       - run: bash <(curl -s https://codecov.io/bash)
         if: ${{ success() }}
+  
+  terraform-plan:
+    runs-on: ubuntu-18.04
+
+    env:
+      ARM_CLIENT_ID: ${{secrets.SP_CLIENT}}
+      ARM_CLIENT_SECRET: ${{secrets.SP_PASSWORD}}
+      ARM_SUBSCRIPTION_ID: ${{secrets.SUBSCRIPTION_ID}}
+      ARM_TENANT_ID: ${{secrets.SP_TENANT}}
+      AZURE_STORAGE_ACCOUNT: "tstate31414"
+      TF_ACTION_WORKING_DIR: "./docker/setup/terraform"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+
+      - name: Terraform Init
+        run: terraform init
+
+      - name: Terraform Format
+        run: terraform fmt -check
```

Comment:
For consistency with the other tooling, I would suggest to add the Terraform format checking to `docker/ci/Dockerfile` which is the container that contains all of the devtools. I would also suggest to add `terraform validate` to the CI suite so that we can automatically detect bad arguments/etc.